lib/WUI/espif.cpp

```diff
@@ -488,6 +505,10 @@ err_t espif_init(struct netif *netif) {
         return ERR_ALREADY;
     }
 
+    struct netif *previous = active_esp_netif.exchange(netif);
+    assert(previous == nullptr);
+    (void)previous; // Avoid warnings in release
```

Comment:
```suggestion
    assert(nullptr == active_esp_netif.exchange(netif));
```

Reply:
Certainly not. That `.exchange` is an important side effect and therefore can't be inside the call to assert. It would be removed in release build.